$NetBSD: patch-bc,v 1.1 2006/10/18 13:11:59 rillig Exp $

Type mismatch, caused by not using header files for function
declarations.

--- unix/x11.cpp.orig	2004-12-30 23:15:47.000000000 +0100
+++ unix/x11.cpp	2006-10-18 14:18:18.126928644 +0200
@@ -209,7 +209,7 @@ int ErrorHandler (Display *, XErrorEvent
 void TVMode (int width, int height);
 void S9xDisplayStateChange (const char *str, bool8 on);
 
-void Init_2xSaI (uint32);
+int Init_2xSaI (uint32);
 void Super2xSaI (uint8 *srcPtr, uint32 srcPitch,
 		 uint8 *deltaPtr, uint8 *dstPtr, uint32 dstPitch,
 		 int width, int height);
